Modules/rat_command.py

```diff
@@ -78,7 +85,18 @@ class NameCollisionException(CommandException):
                 log.debug(
                     f"Rule {getattr(command_fun, '__name__', '')} matching {ctx.words[0]} found.")
             else:
-                log.debug(f"Could not find command or rule for {prefix}{ctx.words[0]}.")
+                # might be a fact
```

Comment:
I question if this handling logic should occur in the facts manager instance, rather than in `rat_command`

This logic really isn't relevant to command handling, and is specific to facts handling therefore should occur inside the facts module

Reply:
the code being there instead of in the facts module reduces (unneeded) nesting. It may be specific to facts, but it is also very specific to figuring out if the invoked "command" is indeed a fact or not.

Reply:
👍